codecov.yml

```
@@ -1,2 +1,10 @@
 codecov:
   branch: main # set new Default branch
+coverage:
+  status:
+    project:
+      default:
+        informational: false
```

```suggestion
        informational: true
```

https://docs.codecov.com/docs/commit-status#informational